.circleci/config.yml

```diff
@@ -61,6 +61,16 @@ jobs:
       - run-bazel-rbe:
           command: bazel test $(bazel query 'kind(checkstyle_test, //...)')
 
+  build-unused-deps:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run:
+          command: bazel run @graknlabs_build_tools//unused_deps -- list
+          no_output_timeout: 30m
```

Comment:
why do we need `no_output_timeout` ?

Reply:
@haikalpribadi essentially, `@graknlabs_build_tools//unused_deps` invokes `bazel build //...` internally which might've taken more than 10 minutes (which is the default timeout)

Reply:
why does it take more than 10 minutes, @vmax ? Is that expected or a problem?

Reply:
it should be less than 1 hour now, given that we no longer need to unpack and re-pack the zip

Reply:
@haikalpribadi i think that is expected for a cold (no cache/RBE involved) build